test/lightning/accounts_test.exs

```diff
@@ -407,24 +406,29 @@ defmodule Lightning.AccountsTest do
           )
         end)
 
-      %{user: user, token: token, email: email, now: now}
+      %{user: user, token: token, email: email}
     end
 
     test "updates the email with a valid token", %{
       user: user,
       token: token,
-      email: email,
-      now: now
+      email: email
     } do
-      assert Accounts.update_user_email(user, token) ==
-               {:ok, %{user | email: email, confirmed_at: now}}
+      now = NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second)
+
+      assert {:ok, changed_user} = Accounts.update_user_email(user, token)
 
-      changed_user = Repo.get!(User, user.id)
       assert changed_user.email != user.email
       assert changed_user.email == email
       assert changed_user.confirmed_at
+      assert changed_user.confirmed_at >= now
```

Comment:
I had to add a comparison here because if the suite is running slowly it's possible for this timestamp to be about a second behind.

The `now` variable used to be in the setup block, but I've moved it into the test block now because I removed one test that was redundant.